src/components/chart/chart.scss

```diff
@@ -1,7 +1,23 @@
 .chart {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  height: 100%;
   padding: 20px;
 
   &__visual {
     margin: 0 auto;
   }
+
+  &__block {
```

Comment:
why do you need 'chart__block'  class?? and then modifier for it?

Reply:
the same behavior for two elements (canvas and empty state)

Reply:
is it better to write separate modulators for canvas and empty-state?